src/librustc/lint/builtin.rs

```diff
@@ -254,6 +254,13 @@ declare_lint! {
     "suggest using `dyn Trait` for trait objects"
 }
 
+declare_lint! {
+    pub FULLY_QUALIFIED_PATH_WITH_MODULE,
```

Comment:
"Fully qualified" is `<Type as Trait>::AssocItem`, `::a::b::c` is "absolute", that's why the feature is called `extern_absolute_paths`.